spi/common/core-spi/src/main/java/org/eclipse/edc/spi/types/domain/edr/EndpointDataAddressConstants.java

```diff
@@ -0,0 +1,71 @@
+/*
+ *  Copyright (c) 2023 Bayerische Motoren Werke Aktiengesellschaft (BMW AG)
+ *
+ *  This program and the accompanying materials are made available under the
+ *  terms of the Apache License, Version 2.0 which is available at
+ *  https://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  SPDX-License-Identifier: Apache-2.0
+ *
+ *  Contributors:
+ *       Bayerische Motoren Werke Aktiengesellschaft (BMW AG) - initial API and implementation
+ *
+ */
+
+package org.eclipse.edc.spi.types.domain.edr;
+
+import org.eclipse.edc.spi.result.Result;
+import org.eclipse.edc.spi.types.domain.DataAddress;
+
+import java.util.Map;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+import static java.lang.String.format;
+
+/**
+ *  Constants for {@link EndpointDataReference} mapping to {@link DataAddress}
+ */
+public class EndpointDataAddressConstants {
```

Comment:
nit: add private default CTor to avoid accidental instantiation. Alternatively, this could be an `interface`.